espnet2/asr/encoder/transformer_encoder.py

```diff
@@ -102,9 +102,10 @@ def __init__(
                 pos_enc_class(output_size, positional_dropout_rate),
             )
         elif input_layer is None:
-            self.embed = torch.nn.Sequential(
-                pos_enc_class(output_size, positional_dropout_rate)
-            )
+            if input_size == output_size:
```

Comment:
What’s this?
Can you add a comment?

Reply:
It's just to handle cases when input and output size is different by simply adding a linear projection to output size. Will add a comment thanks!